src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.AreaCalculator;
+import core.basesyntax.interfaces.Describer;
+
+public abstract class Figure implements Describer, AreaCalculator {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The 'Describer' and 'AreaCalculator' interfaces seem to have different responsibilities, so they should not be implemented together if their methods are not conceptually related.